### PR TITLE
Fix phpDocumentor annotations

### PR DESCRIPTION
**Pull request by Kolyunya**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | 

Fixed some inaccurate annotations.

### CODE REVIEW

**Comment by samdark** [comment]
Merged. Thanks!

**Comment by Kolyunya** [comment]
Glad to be of help!

File: framework/base/Module.php [hunk 1]
```diff
@@ -305,7 +305,7 @@ public function setLayoutPath($path)
     /**
      * Returns current module version.
      * If version is not explicitly set, [[defaultVersion()]] method will be used to determine its value.
-     * @return string the version of this module.
+     * @return string|callable the version of this module.
```
Comment (cebe) [inline framework/base/Module.php]: the return value can not be callable as the callable is used to determine it.

File: framework/base/Component.php [hunk 2]
```diff
@@ -566,7 +566,7 @@ public function getBehavior($name)
 
     /**
      * Returns all behaviors attached to this component.
-     * @return Behavior[] list of behaviors attached to this component
+     * @return Behavior[]|null list of behaviors attached to this component
```
Comment (cebe) [inline framework/base/Component.php]: this is not correct, `ensureBehaviors()` will make sure it is an array.

**Comment by cebe** [comment]
fixed in 6301dae

**Comment by Kolyunya** [comment]
@cebe thanks! Sorry for inconvenience.

**Comment by cebe** [comment]
No problem, these were not obvious.

**Pull request merged**